From: Ludovic Fauvet <etix@l0cal.com>
Subject: Qt: undo the FSC/KDE workaround
 This is a partial revert of the "fix" introduced in
 ffe7367b1672ec72705ec4cb440758ec0c869230.
 .
 If you encounter a visual bug in the fullscreen controller and you are
 using KDE, please upgrade to version 4.6.2 or don't use the Oxygen
 theme.
 .
 Ref: https://bugs.kde.org/show_bug.cgi?id=254473
Applied-Upstream: 1.1.10
Origin: upstream, http://git.videolan.org/?p=vlc%2Fvlc-1.1.git;h=d119e5414b7519894f336180bebfd1f160a9c378
Bug-Ubuntu: https://bugs.launchpad.net/bugs/774581

--- a/modules/gui/qt4/components/controller.cpp
+++ b/modules/gui/qt4/components/controller.cpp
@@ -649,12 +649,7 @@
 
     vout.clear();
 
-#ifdef Q_WS_X11
-    setWindowFlags( Qt::Window | Qt::FramelessWindowHint );
-    setWindowModality( Qt::ApplicationModal );
-#else
     setWindowFlags( Qt::ToolTip );
-#endif
     setMinimumWidth( 600 );
 
     setFrameShape( QFrame::StyledPanel );
